src/App.js

```diff
@@ -9,8 +9,19 @@ export class App extends Component {
     movies: moviesFromServer,
   };
 
-  addMovie = (movie) => {
-    // put your code here
+  addMovie = (title, description, imgUrl, imdbUrl, imdbId) => {
```

Comment:
Можливо, одразу додавати фільм у якості об єкта
```suggestion
  addMovie = (newMovie) => {
```